test/e2e/tests/metrics/errors.spec.js

```diff
@@ -42,6 +42,7 @@ const maskedBackgroundFields = [
   'AppStateController.surveyLinkLastClickedOrClosed',
   'AppStateController.recoveryPhraseReminderLastShown',
   'AppStateController.termsOfUseLastAgreed',
+  'AppStateController.isRampCardClosed',
```

Comment:
Oh, this is why the snapshot is showing it as "string". It's being masked. Masking a property means mapping it to `metamask.[property name]`. We don't need to mask this field.